understanding/accessible-authentication-minimum.html

```diff
@@ -125,203 +122,122 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
-                     provisions are in place that make the code available in the user's clipboard. Evaluating
-                     this criterion therefore only requires verification that the web content does allow
-                     pasting the clipboard content in the related authentication challenge field.
+                  <p>検証コードをセカンダリーデバイスで受信又は生成しなければならないシナリオがある。例えば、ラップトップ上のウェブブラウザで認証する場合、携帯電話に SMS テキストメッセージとして送信される確認コードが必要である。しかし、ほとんどの場合、コードをプライマリーデバイスに直接送信し、コピー＆ペーストできる (例えば、セカンダリーデバイス上のコードをコピーしてプライマリデバイスに電子メールで送信したり、共有のクロスデバイスクリップボードを使用して、セカンダリーデバイス上のコンテンツをコピーすると、そのコンテンツをプライマリーデバイスに貼り付けできる)。コードをセカンダリーデバイスからプライマリーデバイスにシームレスに転送できるかどうかの評価は、この達成基準の<em>範囲外</em>である。このようなタイプのセカンダリーデバイスシステムを使用して認証に依存するウェブコンテンツを評価する目的の場合、利用者のクリップボードでコードを利用できるように準備が整っていることが前提となる。したがって、この基準を評価するには、ウェブコンテンツが関連する認証チャレンジフィールドでクリップボードのコンテンツの貼り付けを許可していることを検証する必要があるだけである。
                   </p>
                   
-                  <p>Note that two-factor systems that do not rely on codes — including hardware authentication
-                     devices (such as YubiKey), secondary applications (either on the same primary device,
-                     or on a secondary device) that expect the user to confirm that it is indeed them trying
-                     to log in, and authentication methods provided by the user's operating system (such
-                     as Windows Hello, or Touch ID/Face ID on macOS and iOS) — are <em>not</em><a href="#"> a cognitive function test</a>.
+                  <p>ハードウェア認証デバイス (YubiKey など)、実際にログインしようとしているのが本人であることを利用者が確認することを期待するセカンダリーアプリケーション (同じプライマリーデバイス上か、セカンダリーデバイス上のいずれか)、利用者のオペレーティングシステムによって提供される認証方法 (Windows Hello、macOS、iOS の Touch ID/Face ID など) など、コードに依存しない 2 要素システムは、<a href="#">認知機能テスト</a><em>ではない</em>ことに注意する。
                   </p>
                   
                </section>
                <section id="object-recognition">
                   
-                  <h3>Object Recognition</h3>
+                  <h3>物体の認識</h3>
                   
-                  <p>If a <a href="https://www.w3.org/TR/turingtest/">CAPTCHA</a> is used as part of an authentication process, there must be a method that does not
-                     include a cognitive function test, unless it meets the exception. If the test is based
-                     on something the website has set such as remembering or transcribing a word, or recognizing
-                     a picture the website provided, that would be a cognitive functional test. Recognizing
-                     objects, or a picture the user has provided is a cognitive function test; however,
-                     it is excepted at the AA level.
+                  <p><a href="https://www.w3.org/TR/turingtest/">CAPTCHA</a> が認証プロセスの一部として使用される場合、例外を満たさない限り、認知機能テストを含まない方法が存在しなければならない。単語を記憶したり転記する、ウェブサイトが提供する画像の認識など、ウェブサイトが設定したものに基づいているテストである場合、それは認知機能テストになる。利用者が提供した物体又は画像の認識は認知機能テストである。ただし、AA レベルでは例外となる。
```

Comment:
ニュアンスが伝わりにくい印象を受けたので、意訳気味になりますが、変えてみました。

```suggestion
                  <p><a href="https://www.w3.org/TR/turingtest/">CAPTCHA</a> が認証プロセスの一部として使用される場合、例外を満たさない限り、認知機能テストを含まない方法が存在しなければならない。単語を記憶したり転記する、ウェブサイトが提供する画像の認識など、ウェブサイトが設定したものに基づいているテストである場合、それは認知機能テストになる。物体、又は利用者が提供した画像の認識も認知機能テストではあるが、AA レベルでは例外となる。
```